docs/UpstreamResourceInterceptor Spec.md

```diff
@@ -0,0 +1,98 @@
+# Admin Web App Resource Mapping Spec
```

Comment:
When I was writing this in I was thinking of it as more of a whiteboard than a permanent piece of documentation. I think we should either clean it up to be a very short document explaining our routes, or delete it.

Reply:
Fair point, I think I'll delete it and bake the routing conversation into the eventual New Module docs.